prism-agent/service/server/src/main/scala/io/iohk/atala/verification/controller/http/VcVerificationRequest.scala

```diff
@@ -0,0 +1,110 @@
+  case AudienceCheck extends VcVerification("AudienceCheck")
+  case SubjectVerification extends VcVerification("SubjectVerification")
+  case IntegrityOfClaims extends VcVerification("IntegrityOfClaims")
+  case ComplianceWithStandards extends VcVerification("ComplianceWithStandards")
+  case RevocationCheck extends VcVerification("RevocationCheck")
+  case AlgorithmVerification extends VcVerification("AlgorithmVerification")
+  case SchemaCheck extends VcVerification("SchemaCheck")
+  case SemanticCheckOfClaims extends VcVerification("SemanticCheckOfClaims")
+}
+
+object VcVerificationRequests {
+  object annotations {
+
+    object credential
+        extends Annotation[String](
+          description = "Encoded Verifiable Credential to verify",
+          example =
+            "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
+        )
+    object vcVerification
+        extends Annotation[List[VcVerification]](
+          description = "The list of Verifications to verify. All verifications run if Verifications left empty",
+          example = List(
+            VcVerification.SignatureVerification,
+            VcVerification.IssuerIdentification,
+            VcVerification.ExpirationCheck,
+            VcVerification.NotBeforeCheck,
+            VcVerification.AudienceCheck,
+            VcVerification.SubjectVerification,
+            VcVerification.IntegrityOfClaims,
+            VcVerification.ComplianceWithStandards,
+            VcVerification.RevocationCheck,
+            VcVerification.AlgorithmVerification,
+            VcVerification.SchemaCheck,
+            VcVerification.SemanticCheckOfClaims,
+          )
+        )
+    object vcVerificationRequests
+        extends Annotation[List[VcVerificationRequest]](
+          description = "A list of VcVerificationRequest",
+          example = List(
+            VcVerificationRequest(
+              "joxNbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c",
+              List.empty
+            ),
+            VcVerificationRequest(
+              "3ODkbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c",
+              List(VcVerification.IssuerIdentification)
+            ),
+            VcVerificationRequest(
+              "pXVCbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c",
+              List(VcVerification.RevocationCheck)
+            )
+          )
+        )
+
+  }
+
+  given verificationEncoder: JsonEncoder[VcVerification] =
+    DeriveJsonEncoder.gen[VcVerification]
+
+  given verificationDecoder: JsonDecoder[VcVerification] =
+    DeriveJsonDecoder.gen[VcVerification]
+
+  given verificationSchema: Schema[VcVerification] = Schema.derived
```

Comment:
Let's use `Schema.derivedEnumeration.defaultStringBased` instead, so `VcVerification` is correctly translated to an OAS enumeration and not a `oneOf`.

Reply:
fixed